src/test/java/com/redhat/cloud/notifications/routers/NotificationServiceTest.java

```diff
@@ -50,36 +47,32 @@
      * here. The deserialization is still performed only to verify that the JSON responses data structure is correct.
      */
 
+    private static final String TENANT = "NotificationServiceTest";
+    private static final String USERNAME = "user";
+
     @MockServerConfig
     MockServerClientConfig mockServerConfig;
 
     @Inject
     ResourceHelpers helpers;
 
-    private Header identityHeader;
-
     @BeforeEach
     void beforeEach() {
         RestAssured.basePath = TestConstants.API_NOTIFICATIONS_V_1_0;
-
-        // Add mock-rbac-full access before each test, as some will clear this out,
-        // as they only need partial access
-        String tenant = "NotificationServiceTest";
-        String userName = "user";
-        String identityHeaderValue = TestHelpers.encodeIdentityInfo(tenant, userName);
-        identityHeader = TestHelpers.createIdentityHeader(identityHeaderValue);
-        mockServerConfig.addMockRbacAccess(identityHeaderValue, MockServerClientConfig.RbacAccess.FULL_ACCESS);
-
+        mockServerConfig.clearRbac();
     }
 
-    @BeforeAll
-    @ActivateRequestContext
-    void init() {
-        helpers.createTestAppAndEventTypes();
+    private Header initRbacMock(String tenant, String username, RbacAccess access) {
```

Comment:
I'll probably put that method into some kind of helper later. This code is duplicated in many test classes. But that's out of that PR scope.